content/projects/angular-mocking/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Mocking with Angular
+ready: True
+prerequisites:
+  hard: ["topics/angular-testing-cucumber",
+         "topics/angular-mocking"]
+  soft: []
+  available_options: 
```

Comment:
indentation matters in yaml.

There should be no spaces in front of `available_options` or `submission_type`. 

This indentation says that `available_options` and `submission_type` are inside prerequisites which is incorrect. Yaml is a data structure that gets navigated.